src/utils/nanvixd/src/args.rs

```diff
@@ -7,6 +7,11 @@
 
 use crate::config;
 use ::anyhow::Result;
+use hwloc::HwLoc;
+use std::{
```

Comment:
(nitpick) Could you prefix with `::` to be consistent with the project guidelines?